fixes/github-issue-create.js

```diff
@@ -47,31 +47,45 @@ async function createGithubIssue(fs, options, targets, dryRun = false) {
       this.Octokit
     )
 
+    let assignees = []
+    const assigneeSelectCount = 0
     // Retrieve committers of a repository if option is enabled
```

Comment:
Retrieve top committers of a repository if assignTopCommitter option is set or undefined.

Reply:
Thanks!